cloudcasa_image_version

```diff
@@ -1 +1 @@
-2.37.0-5
+v2.37.0.6
```

Comment:
@jongarner, is it okay to change the image tag like this? I noticed the old one has this strange format, so the new one matches the format we use for other forks.

Reply:
Great. Thanks!